### ISSUE TITLE
Goto next/previous chapter

### ISSUE THREAD

**Issue by DKami** [EmmaLa/MangaReader#1]
Greetings,

I think it could be nice if we could go to the next or previous chapter when we're already reading one chapter.

**Comment by EmmaLa** [comment]
Yeah of course i think i'll add this functionnality soon or if you want to make a pull request i'll note refuse.

**Comment by DKami** [comment]
Ok buddy, I'll probably work on this, but I've never worked on reactJS.

**Comment by EmmaLa** [comment]
You can see some tutorial on internet to get started with, if you are inspired to do that i'll let you do it otherwise i'll do it.
